tests/contrib/test_caching.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from django.core.cache import cache
+from django.urls import reverse
+
+from adhocracy4.projects.enums import Access
+from adhocracy4.test.factories import ProjectFactory
+from meinberlin.apps.contrib import caching
+from meinberlin.test.factories.extprojects import ExternalProjectFactory
+from meinberlin.test.factories.plans import PlanFactory
+from meinberlin.test.factories.projectcontainers import ProjectContainerFactory
+
+
+def test_cache_delete():
+    ...
+
+
+def test_cache_add_or_query():
+    ...
+
+
+def test_cache_add_or_serialize():
+    ...
```

Comment:
the delete cache you can include directly in the test_list_url.
the other two are indirectly tested with the GET call in the same test, so no need to expand.